### PR TITLE
Added a few smileys -> emoji combinations

### PR DESCRIPTION
**Pull request by SpaceCafe**

### CODE REVIEW

**Comment by d9h20f** [comment]
You're changing a whole bunch of different emoticons to the same emoji. I don't think ":3" and "XD" look anything like, but you've made them into the same "😁" face - which itself doesn't look like either of the aforementioned emoticons.

**Comment by sbernard31** [comment]
XD is more = 😆, 
:3 [maybe a cat face](https://www.quora.com/What-does-the-emoticon-3-mean?share=1) : 😺 ?

more example : https://developer.gnome.org/gtk3/stable/GtkIMContextSimple.html

File: main/src/ui/chat_input/smiley_converter.vala [hunk 1]
```diff
@@ -13,18 +13,41 @@ class SmileyConverter {
...
+        smiley_translations[":-("] = "🙁";
         smiley_translations[":'("] = "😢";
+        smiley_translations[":'-("] = "😢";
+        smiley_translations[":|"] = "😐";
+        smiley_translations[":-|"] = "😐";
         smiley_translations[":/"] = "😕";
+        smiley_translations[":-/"] = "😕";
+        smiley_translations[":*"] = "😘";
+        smiley_translations[":-*"] = "😘";
+        smiley_translations["<3"] = "💓";
```
Comment (dos1) [inline main/src/ui/chat_input/smiley_converter.vala]: Shouldn't it rather be ❤?

**Comment by fiaxh** [comment]
This PR is similar to #42:

> For now the list will stay as it is, containing only the smileys with the most obvious unicode counterpart. If everyone just adds the replacements they would like to have, in the end this feature will become unsuitable for everyone.

**Pull request closed without merging**